src/project-css/resources/views/user/data.blade.php

```diff
@@ -17,18 +18,23 @@
   <div class="container">
     <div class="row">
         <div class="col-xs-12 col-sm-12 col-md-12">
-          <form class="form-horizontal" role="form" method="POST" action="{{ url('/login') }}">
+          <form class="form-horizontal" role="search" method="get" action="{{ url('/data',$tblId) }}">
               {{ csrf_field() }}
-
               <div class="form-group{{ $errors->has('search') ? ' has-error' : '' }}">
                   <div class="col-md-10 col-sm-10 col-xs-8">
-                      <input id="search" type="search" class="form-control searchBar" name="search" value="{{ old('search') }}" required autofocus>
+                    <select id="tblCol" name="tblCol" class="form-control">
```

Comment:
All input elements need a label tag, for our accessibility compliance, this needs to happen in all views.